pkg/cloud/services/networking/router.go

```diff
@@ -196,10 +202,11 @@ func (s *Service) DeleteRouter(openStackCluster *infrav1.OpenStackCluster, clust
 	}
 
 	if subnet.ID != "" {
+		mc := metrics.NewMetricPrometheusContext("attach_interface", "delete")
```

Comment:
Thank you very much for adding these. 
I'm not sure which tag to use:
* attach_interface, create/delete, or
* router_interface, add/remove, or
* router, update

The second would probably align best with https://github.com/gophercloud/gophercloud/blob/master/openstack/networking/v2/extensions/layer3/routers/urls.go. @chrischdi WDYT?

Reply:
agree attach_interface is bad naming
seems `interface` is good alternative?

Reply:
I would prefer `router_interface`. @sbueringer @chrischdi

Reply:
I'd prefer `router_interface`. I would **not** use `router` `update` because this is a whole different api call.

Reply:
and `router` in `router_interface` because we are using the routers package here :-)

Reply:
I would also use `router_interface`